etl/controller/controller.go

```diff
@@ -53,7 +53,12 @@ func NewController(cfg configuration.Controller, extractor interfaces.JetDropsEx
 		return nil, errors.Wrap(err, "can't get not complete pulses from storage")
 	}
 	for _, p := range pulses {
-		key := types.Pulse{PulseNo: p.PulseNumber}
+		key := types.Pulse{PulseNo: p.PulseNumber, PrevPulseNumber: p.PrevPulseNumber, NextPulseNumber: p.NextPulseNumber}
+		func() {
+			c.jetDropRegisterLock.Lock()
```

Comment:
Is it bugfix? i can't find previous place where you created it.

Reply:
Yes, if we save pulse and don't save any jetDrops for it, we didn't add it to `jetDropRegister` and data for it didn't reload. Now it fixed.